titanic.ipynb

```diff
@@ -5,14 +5,14 @@
       "cell_type": "markdown",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 1 Line 1_
```markdown
## Titanic Dataset
```
markdown comment

_Respond and view the context [here](https://gitnotebooks.com/smith-kyle/notebook-diff-tests/pull/25)._